adserver/forms.py

```diff
@@ -392,3 +394,39 @@ class Meta:
             "allow_house_campaigns",
             "record_placements",
         ]
+
+
+class InviteUserForm(forms.ModelForm):
+
+    """Used to invite users to collaborate on an advertiser/publisher."""
```

Comment:
Did we not build this for publishers yet?

Reply:
No. I didn't. I haven't gotten many requests to do that. Have you?

Reply:
Nope, and if so I can always just add them in the admin.

Reply:
My plan was to allow a view for a publisher to re-use the form. The view can't be re-used exactly although maybe refactored into a mixin.